pkgs/by-name/sw/sway-easyfocus/package.nix

```diff
@@ -15,16 +15,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "sway-easyfocus";
-  version = "unstable-2023-11-05";
+  version = "unstable-2025-01-14";
```

Comment:
This version should technically be `0.2.0-unstable-2025-01-15` as per https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

Reply:
done